examples/bri-3/src/bri/transactions/agents/transactions.agent.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  BadRequestException,
+  Injectable,
+  NotFoundException,
+} from '@nestjs/common';
+import { Transaction } from '../models/transaction';
+import { TransactionStatus } from '../models/transactionStatus.enum';
+
+import {
+  DELETE_WRONG_STATUS_ERR_MESSAGE,
+  NOT_FOUND_ERR_MESSAGE,
+  UPDATE_WRONG_STATUS_ERR_MESSAGE,
+} from '../api/err.messages';
+import { TransactionStorageAgent } from './transactionStorage.agent';
+import { BpiAccount } from '../../identity/bpiAccounts/models/bpiAccount';
+
+@Injectable()
+export class TransactionAgent {
+  constructor(private repo: TransactionStorageAgent) {}
+  public throwIfCreateTransactionInputInvalid() {
```

Comment:
we dont have interface or anything enforcing empty implementations like this, should we add them?

Reply:
Could you please elaborate your reasoning. I just added this as a placeholder.

Reply:
@ognjenkurtic it might be prudent to indicate that this is a placeholder and add a TODO to ensure proper input validation in the interface or class.

Reply:
added TODO